src/tasks/Crossgen2Tasks/Microsoft.NET.CrossGen.targets

```diff
@@ -443,7 +443,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     <ResolveReadyToRunCompilers RuntimePacks="@(ResolvedRuntimePack)"
                                 Crossgen2Packs="@(ResolvedCrossgen2Pack)"
                                 TargetingPacks="@(ResolvedTargetingPack)"
-                                RuntimeGraphPath="$(BundledRuntimeIdentifierGraphFile)"
+                                RuntimeGraphPath="$([MSBuild]::ValueOrDefault('$(RuntimeIdentifierGraphPath)', '$(BundledRuntimeIdentifierGraphFile)'))"
```

Comment:
Another place using `BundledRuntimeIdentifierGraphFile` is in pretest.proj: https://github.com/dotnet/runtime/blob/f8d9b3cf5b5bc7439f4bf631c433e26ebfe49c06/src/libraries/pretest.proj#L105 Can we update it as well?

Reply:
Opened https://github.com/dotnet/runtime/pull/91102